fiberassign Makefile broken?

PR #104 appears to have broken "make install" for the fiberassign master module on cori.  What I used to do (I think) as the desi user:
```
source /project/projectdirs/desi/software/desi_environment.sh master
/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master
git pull
make install
```
That now complains
```
Makefile:60: platforms/nesrc_cori_haswell: No such file or directory
make: Entering directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master'
make: *** No rule to make target 'platforms/nesrc_cori_haswell'.  Stop.
make: Leaving directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master'
```
Switching to `make PLATFORM=harpconfig install` gets further but still fails to install the fiberassign executable:
```
make: Entering directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master'
make[1]: Entering directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master/src'
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o fiberassign.o -c fiberassign.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o collision.o -c collision.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o feat.o -c feat.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o global.o -c global.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o misc.o -c misc.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o structs.o -c structs.cpp
g++  -O3 -fPIC -pthread -fPIC -DPIC -fopenmp -fopenmp -I.  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include  -I/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/include -o version.o -c version.cpp
g++ -o fiberassign fiberassign.o collision.o feat.o global.o misc.o structs.o version.o -fopenmp  -L/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -L/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -Wl,-R,/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -L/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -Wl,-R,/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib  -L/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -Wl,-R,/global/common/cori/contrib/desi/code/desiconda/20170613-1.1.4-spectro_aux/lib -lcfitsio -lm
make[1]: Leaving directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master/src'
/bin/mkdir -p /global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master
Makefile:72: recipe for target 'install' failed
make: *** [install] Error 1
make: Leaving directory '/global/common/cori/contrib/desi/desiconda/20170613-1.1.4-spectro/code/fiberassign/master'
```
What should be done to compile and install this?

I also tried this on edison using the 18.2 environment, with the same failure:
```
[edison12 ~] source /project/projectdirs/desi/software/desi_environment_18.2.sh
[edison12 ~] module unload fiberassign
[edison12 ~] cd $SCRATCH/desi/code/fiberassign
[edison12 fiberassign] make PLATFORM=nersc_edison install
make: Entering directory '/scratch2/scratchdirs/sjbailey/desi/code/fiberassign'
make[1]: Entering directory '/scratch2/scratchdirs/sjbailey/desi/code/fiberassign/src'
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o fiberassign.o -c fiberassign.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o collision.o -c collision.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o feat.o -c feat.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o global.o -c global.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o misc.o -c misc.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o structs.o -c structs.cpp
g++ -O3 -std=c++11 -fPIC -DNDEBUG -fopenmp -I. -I/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/include -o version.o -c version.cpp
g++ -o fiberassign fiberassign.o collision.o feat.o global.o misc.o structs.o version.o -fopenmp -L/global/common/software/desi/edison/desiconda/20180130-1.2.4-spec/aux/lib -lcfitsio -lm
make[1]: Leaving directory '/scratch2/scratchdirs/sjbailey/desi/code/fiberassign/src'
/bin/mkdir -p /scratch2/scratchdirs/sjbailey/desi/code/fiberassign
Makefile:72: recipe for target 'install' failed
make: *** [install] Error 1
make: Leaving directory '/scratch2/scratchdirs/sjbailey/desi/code/fiberassign'
```

--- a/Makefile
+++ b/Makefile
@@ -70,7 +70,7 @@ all :
 
 install : all
 	- /bin/mkdir -p $(INSTALL_DIR)
-	@ for d in bin py script test; do test ! -d $(INSTALL_DIR)/$$d && /bin/cp -a $$d $(INSTALL_DIR) ; done
+	@ for d in bin py script test; do if [ ! -d $(INSTALL_DIR)/$$d ]; then /bin/cp -a $$d $(INSTALL_DIR) ; fi ; done
 	@ for f in $(SUBDIRS); do $(MAKE) -C $$f install ; done
 
 clean :

--- a/Makefile
+++ b/Makefile
@@ -40,7 +40,7 @@ ifeq ($(NERSC_HOST), edison)
 PLATFORM := nersc_edison
 endif
 ifeq ($(NERSC_HOST), cori)
-PLATFORM := nesrc_cori_haswell
+PLATFORM := nersc_cori_haswell
 endif
 
 else

